x-pack/plugins/cases/public/components/user_actions/comment/registered_attachments.tsx

```diff
@@ -50,7 +50,7 @@ type BuilderArgs<C, R> = Pick<
 /**
  * Provides a render function for attachment type
  */
-const getAttachmentRenderer = memoize(() => {
+const getAttachmentRenderer = memoize((cachingKey: string) => {
```

Comment:
Is `cachingKey` used in this function?

Reply:
Done https://github.com/elastic/kibana/pull/158441/commits/8c7db6e019dab554f6d5715d463bba599af11e49. Can you please test carefully?